packages/documentation/src/content/docs/telemetry/overview.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Overview
+---
+
+## Purpose
+
+The objective of the telemetry feature is to gather metrics and establish an infrastructure for visualizing valuable network insights. The metrics we collect include:
+
+- The total amount of money transferred via packet data within a specified time frame (daily, weekly, monthly).
+- The number of transactions from outgoing payments that have been at least partially successful.
+- The average amount of money held within the network per transaction.
+
+Our goal is to use these data for our own insights and to enable [Account Servicing Entities](/reference/glossary#account-servicing-entity) (ASEs) to gain their own insights. We aim to track the growth of the network in terms of transaction sizes and the number of transactions processed.
```

Comment:
Maybe the "we aim to track the growth of the network..." sentence should go first?